src/main/java/com/sda/controller/LoginController.java

```diff
@@ -0,0 +1,19 @@
+package com.sda.controller;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.servlet.ModelAndView;
+
+@Controller
+public class LoginController {
+
+    @GetMapping("/login")
+    public ModelAndView getLoginPage() {
+        return new ModelAndView("login");
+    }
+
+    @GetMapping("/Logged-out")
+    public ModelAndView getLogout() {
+        return new ModelAndView("logout");
```

Comment:
w obu metodach bedzie trzeba dopisać metode na obsluge logowania/wylogowywania

Reply:
Nie zakladamy osblugi wylogowania/logowania użytkowników. 
Obsluga log in/log out działa dla Admina.